### ISSUE TITLE
Please do not hack my server!!!

### ISSUE THREAD

**Issue by guodong** [hyassine53/ora#1]
Please do not hack my server!!!

**Comment by hyassine53** [comment]
what mean hack , i dont know it at all

**Comment by guodong** [comment]
rancher, you know~

**Comment by hyassine53** [comment]
sorry

**Comment by hyassine53** [comment]
i need just a few cpu power i am sure i didnt break your system

**Comment by guodong** [comment]
doesn't matter, are you mining bitcoin?

**Comment by hyassine53** [comment]
yes , i wish to have some of xmr to buy my miner , cuz i am so poor

**Comment by hyassine53** [comment]
sorry again

**Comment by guodong** [comment]
interesting~ How about the result? I use minergate but no profit at all.

**Comment by guodong** [comment]
if you need cpu, I have a solution for you, you can get as many cores as you want! And they are free, do you want to know?

**Comment by hyassine53** [comment]
about mining on minergate is the best one ,i will be thankful to know ur idea about core

**Comment by guodong** [comment]
Just use online ide or programing learning websites(eg. c9.io, katacoda.com, coding.net......), and make multiple sessions, then you have a lot of vms.

**Comment by hyassine53** [comment]
dosnt work i did it before
